### PR TITLE
docs: Updated ReadMe's Description.

### PR DESCRIPTION
**Pull request by abha224**
### Description

Readme.md file is updated with relevant description.

Fixes # [14](https://github.com/anitab-org/anitab-org.github.io/issues/14#issue-602851768)

### Type of Change:
- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by abha224** [comment]
@annabauza I have updated the README file. 
I have not added the contributing guidelines link as we dont have the doc in the repo yet. However i had seen an issue created for including the doc. So will it be ok if the link is updated once the doc is made, or its better to eliminate it for the time being? :sweat_smile: 
Please let me know if any changes need to be made, thank you!

**Comment by abha224** [comment]
@mayburgos Thank you for the review! Ive made the changes you suggested and updated all links.

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+
+## Contributing
+
+**This project is under active development**
+
+Please read our [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/code_of_conduct.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/reporting_guidelines.md) thoroughly.
+
+## Branches
+
+* **master** This contains the code which will be deployed into production.
```
Comment (annabauza) [inline README.md:14]: This contains a deployment of web along with `gh-pages` branch, `production` is a branch that triggers deployment.

Comment (abha224) [reply README.md:14]: Made the necessary changes, pls do take a look. :sweat_smile:

Comment (annabauza) [reply README.md:14]: * **master** and **gh-pages** These branches contains the deployment of the website. 
* **develop** This contains the latest code. All the contributing PRs must be sent to this branch.
* **production** Merge develop into production triggers deployment of  the website.

Comment (abha224) [reply README.md]: done 👍

**Pull request merged**